lldb/test/API/lang/swift/macro/Makefile

```diff
@@ -0,0 +1,22 @@
+SWIFT_SOURCES := main.swift
+SWIFTFLAGS_EXTRAS = -load-plugin-library $(BUILDDIR)/libMacro.dylib \
+                    -enable-experimental-feature Macros
+SWIFT_HOST_DIR = $(shell dirname $(SWIFTC))/../lib/swift/host
+
+all: libMacro.dylib $(EXE)
+
+include Makefile.rules
+
+libMacro.dylib:
+	$(MAKE) MAKE_DSYM=$(MAKE_DSYM) CC=$(CC) SWIFTC=$(SWIFTC) \
+		ARCH=$(ARCH) DSYMUTIL=$(DSYMUTIL) \
+		VPATH=$(SRCDIR) -I $(SRCDIR) \
+		-f $(THIS_FILE_DIR)/Makefile.rules \
+		DYLIB_SWIFT_SOURCES=macro.swift \
+		DYLIB_NAME=Macro \
+		DYLIB_ONLY=YES \
+		SWIFTFLAGS_EXTRAS="-I $(SWIFT_HOST_DIR)" \
+		LD_EXTRAS="-L$(SWIFT_HOST_DIR) -lSwiftSyntax -lSwiftSyntaxBuilder" \
```

Comment:
Was the `-lSwiftSyntax -lSwiftSyntaxBuilder` necessary? It seems like it should be handled by auto linking, and if that wasn't working, you'd need to add `-lSwiftSyntaxMacros` and `-lSwiftDiagnostics` and several others from the swift-syntax package manually.